README.md

```diff
@@ -28,27 +25,30 @@ Once I get around to it, I hope to:
 - [ ] Documentation generator in markdown?
 - [ ] Syntax highlighted source code output?
 
-Requirements
-------------
+## Requirements
 
-* [lark](https://github.com/lark-parser/lark) (for grammar-based parsing)
-* [pytmc](https://github.com/pcdshub/pytmc) (for directly loading TwinCAT projects)
+- [lark](https://github.com/lark-parser/lark) (for grammar-based parsing)
+- [pytmc](https://github.com/pcdshub/pytmc) (for directly loading TwinCAT projects)
 
-How to use it
--------------
+## Installation
 
-1. Preferably using non-system Python, set up an environment using, e.g., miniconda:
-```bash
-$ conda create -n blark-env python=3.7
-$ conda activate blark-env
-```
-
-2. Install the library (using conda or otherwise, these steps are the same)
+Installation is quick with Pip.
 
 ```bash
-$ pip install blark
+pip install --upgrade blark
 ```
 
+## Quickstart
+
+1. Preferably using non-system Python, set up an environment using, e.g., miniconda:
+  ```bash
+  $ conda create -n blark-env python=3.7
```

Comment:
Looks like I added blark to conda-forge since I last updated the README (ages ago...).

This step on conda can now be done in a single go:
```suggestion
  $ conda create -n blark-env -c conda-forge python=3.7 blark
```

Which means that the `pip install` is only necessary for virtualenv / system python / non-conda environments.

I'd say let's break this up into two sections:
1. Here's how to install blark with conda (using the line above)
2. Here's how to install blark in a virtualenv (using standard library venv: [guide](https://realpython.com/python-virtual-environments-a-primer/))